sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Features Added
 
-### Breaking Changes
+### Breaking Changes from 1.5.0-beta.2
```

Comment:
This might not get picked up by the release notes automation. Does it look a specific header or just the beginning of it @weshaggard?

Reply:
For the sections parsing it looks for a particular header. We should add the versioning info as content under this header.

Reply:
Okay, I will update it in Release Prep PR.